src/client/components/modes/modes.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import './modes.css';
+import ModesItem from '../modesItems/modesItem';
+import imageArray from '../../helpers/constants';
+
+const Modes = () => {
+  const fullModesItem = imageArray.map((item, index) => <ModesItem key={index} iconUrl={item.img} text={item.text} />);
```

Comment:
You can destruct item parameter that map function accepts, like this 
map(({  img, text }, index) =>